sympy/utilities/tests/test_lambdify.py

```diff
@@ -829,3 +830,15 @@ def test_lambdify_dummy_arg():
     d2 = Dummy('x')
     f2 = lambdify(d2, d2 + 1)
     assert f2(2) == 3
+
+def test_issue_14283():
+    if not numpy:
+        skip("numpy not installed.")
+    prntr = PythonCodePrinter()
+    expr1 = zoo
+    func = lambdify(x, expr1, "numpy")
+    assert prntr.doprint(func(1)) == "float('nan')"
+
+    expr2 = -oo
+    func = lambdify(x, expr2, "numpy")
+    assert prntr.doprint(func(1)) == '-inf'
```

Comment:
this test could be changed into:
```
assert prntr.doprint(-oo) == 'inf'
```
the same transformation (dropping lambdify) also goes for the test before it.

Reply:
Thank you @bjodah.
The changes have been made.